app/src/main/res/layout/fragment_login.xml

```diff
@@ -0,0 +1,110 @@
+            android:layout_height="193dp"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            android:alpha="0.9"
+            android:background="@drawable/gradient" />
+
+        <EditText
+            android:id="@+id/vLoginEmailEditText"
+            android:layout_width="match_parent"
+            android:layout_height="56dp"
+            android:layout_marginStart="16dp"
+            android:layout_marginTop="16dp"
+            android:layout_marginEnd="16dp"
+            android:ems="10"
+            android:hint="@string/fragment_login_email_hint_edit_text"
+            android:inputType="textEmailAddress"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/vLoginWoloxLogoImageView" />
+
+        <ImageView
+            android:id="@+id/vLoginWoloxLogoImageView"
+            android:layout_width="147dp"
+            android:layout_height="122dp"
+            android:layout_marginStart="8dp"
+            android:layout_marginTop="124dp"
+            android:layout_marginEnd="8dp"
+            android:contentDescription="@string/fragment_login_wolox_logo_description"
+            android:src="@drawable/wolox_logo_large"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="@+id/vLoginBannerImageView" />
+
+        <EditText
+            android:id="@+id/vLoginPasswordEditText"
+            android:layout_width="match_parent"
+            android:layout_height="56dp"
+            android:layout_marginStart="16dp"
+            android:layout_marginTop="8dp"
+            android:layout_marginEnd="16dp"
+            android:ems="10"
+            android:hint="@string/fragment_login_password_hint"
+            android:inputType="textPassword"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/vLoginEmailEditText" />
+
+        <Button
+            android:id="@+id/vLoginLogInButton"
+            android:layout_width="match_parent"
+            android:layout_height="56dp"
+            android:layout_marginTop="24dp"
+            android:layout_marginStart="16dp"
+            android:layout_marginEnd="16dp"
+            android:text="@string/fragment_login_login_button"
+            app:layout_constraintTop_toBottomOf="@+id/vLoginPasswordEditText"
+            tools:layout_editor_absoluteX="23dp" />
+
+        <Button
+            android:id="@+id/vLoginSignUpButton"
+            android:layout_width="match_parent"
+            android:layout_height="56dp"
+            android:layout_marginTop="8dp"
+            android:background="@color/colorWoloxBlue"
```

Comment:
We can group size attributes together and leave text, textColor and backgroun below

Reply:
✔ **Done**